src/types/component.go

```diff
@@ -32,7 +32,7 @@ type ZarfComponent struct {
 	Group string `json:"group,omitempty" jsonschema:"description=Create a user selector field based on all components in the same group"`
 
 	//Path to cosign publickey for signed online resources
-	CosignKeyPath string `json:"cosignKeyPath,omitempty" jsonschema:"description=Specify a path to a public key to validate signed online resources"`
+	CosignKeyPath string `json:"cosignKeyPath,omitempty" jsonschema:"description=[Deprecated] Specify a path to a public key to validate signed online resources"`
```

Comment:
`import` will need a `public-key` for `url` imports. As it currently stands, you will not be able to import a component from a skeleton package that has been signed with a different key than the one you are creating a package with.

Reply:
I would prefer to do that in a separate PR.

Reply:
Made this issue to make sure this thought does not get lost. 

https://github.com/defenseunicorns/zarf/issues/1920